design-editor/src/panel/property/component/component-element.js

```diff
@@ -85,76 +84,72 @@ class Component extends DressElement {
         this._unsetDraggable(this._$componentButtonList);
     }
 
-    /**
-     * Build main DOM structure and initialize all properties
-     * @private
-     */
-    _initialize() {
-        var self = this,
-            componentsContainers = {},
-            componentsChildren = {};
-
-        self._$componentButtonList = null;
-        self._componentPackage = packageManager.getPackages(Package.TYPE.COMPONENT);
-        componentsInfo = self._componentPackage._packages;
-
-        Object.keys(componentsInfo).forEach((componentName) => {
-            var componentOptions = componentsInfo[componentName].options;
-
-            if (componentOptions.constraint) {
-                componentOptions.constraint.forEach((constraint) => {
-                    if (componentsContainers[constraint]) {
-                        componentsContainers[constraint].push(componentName);
-                    } else {
-                        componentsContainers[constraint] = [componentName];
-                    }
-                    if (componentsChildren[componentName]) {
-                        componentsChildren[componentName].push(constraint);
-                    } else {
-                        componentsChildren[componentName] = [constraint];
-                    }
-                });
-            }
-        });
-
-        self._componentsContainers = componentsContainers;
-        self._componentsChildren = componentsChildren;
-
-        $.get(path.join(AppManager.getAppPath().src, TEMPLATE_FILE_PATH, TEMPLATE_FILE_NAME), (templateString, err) => {
-            if (templateString) {
-                self.$el.append(mustache.render(templateString));
-                self._render();
-            } else {
-                throw new Error(err);
-            }
-        });
-    }
+	/**
+	 * Build main DOM structure and initialize all properties
+	 * @private
+	 */
+	_initialize() {
+		const self = this,
```

Comment:
Please unify your approach while using self or this not both in the same function scope